ACPs/224-dynamic-gas-limit-in-subnet-evm/README.md

```diff
@@ -0,0 +1,233 @@
+| M | minimum gas price | 1∗10^−18 AVAX |
+| K | initial gas price update factor | 87*T |
+
+### Prior Subnet-EVM Fee Configuration Parameters
+
+Prior to this ACP, the Subnet-EVM fee configuration and fee manager precompile used the following parameters to control the fee mechanism:
+
+**GasLimit**:
+  Sets the max amount of gas consumed per block.
+
+**TargetBlockRate**:
+  Sets the target rate of block production in seconds used for fee adjustments. If the actual block rate is faster than this target, block gas cost will be increased, and vice versa.
+
+**MinBaseFee**:
+  The minimum base fee sets a lower bound on the EIP-1559 base fee of a block. Since the block's base fee sets the minimum gas price for any transaction included in that block, this effectively sets a minimum gas price for any transaction.
+
+**TargetGas**:
+  Specifies the targeted amount of gas (including block gas cost) to consume within a rolling 10s window. When the dynamic fee algorithm observes that network activity is above/below the `TargetGas`, it increases/decreases the base fee proportionally to how far above/below the target actual network activity is.
+
+**BaseFeeChangeDenominator**:
+  Divides the difference between actual and target utilization to determine how much to increase/decrease the base fee. A larger denominator indicates a slower changing, stickier base fee, while a lower denominator allows the base fee to adjust more quickly.
+
+**MinBlockGasCost**:
+  Sets the minimum amount of gas to charge for the production of a block.
+
+**MaxBlockGasCost**:
+  Sets the maximum amount of gas to charge for the production of a block.
+
+**BlockGasCostStep**:
+  Determines how much to increase/decrease the block gas cost depending on the amount of time elapsed since the previous block. If the block is produced at the target rate, the block gas cost will stay the same as the block gas cost for the parent block. If it is produced faster/slower, the block gas cost will be increased/decreased by the step value for each second faster/slower than the target block rate accordingly.
+  Note: if the `BlockGasCostStep` is set to a very large number, it effectively requires block production to go no faster than the `TargetBlockRate`.
+  Ex: if a block is produced two seconds faster than the target block rate, the block gas cost will increase by `2 * BlockGasCostStep`.
+
+### ACP-176 Parameters in Subnet-EVM
+
+ACP-176 will make `GasLimit` and `BaseFeeChangeDenominator` configurations obsolete in Subnet-EVM.
+
+`TargetBlockRate`, `MinBlockGasCost`, `MaxBlockGasCost`, and `BlockGasCostStep` will be kept same because ACP-176 still uses block gas cost to control the block production rate and surcharge for producing a block faster than the target rate. Subnet-EVM is configured to use following default values, which will be kept same:
+- `TargetBlockRate`: 2 (seconds)
+- `MinBlockGasCost`: 0 (gas)
+- `MaxBlockGasCost`: 1,000,000 (gas)
+- `BlockGasCostStep`: 200,000 (gas)
```

Comment:
Are these parameters formulaicly hardcoded or defining as hard coded variables somewhere? Understand that they are not included in the genesis config example but am unsure where or how these values could be returned